### DISCUSSION TITLE
Linear vs Inverse perpetual symbols

### DISCUSSION THREAD

**Discussion by docek** [bmoscon/cryptofeed#803] [Q&A] [answered]
Hi, some exchanges like Phemex has both inverse perpetual swap and linear perpetual swap. However I can not find a way how to use both in Cryptofeed since BTC-USD-PERP gives me just the linear perpetual swap on Phemex. Am I missing something or it is not possible to get inverse perpetual swap?

**Comment by bmoscon** [comment, marked as answer]
support needs to be added to the library
